pupil-spa/src/app/services/storage/storage.service.spec.ts

```diff
@@ -166,4 +168,47 @@ describe('StorageService', () => {
       expect(localStorageItems[items[2].key]).toEqual('foo-bar');
     });
   });
+  describe('set multiple items in local storage', () => {
+    it('saves 10000 items in local storage', () => {
+      for (let i = 0; i < 50000; i++) {
```

Comment:
what are we testing here, the limits of localStorage?  these are being added directly to local storage without going through any of our services.

Reply:
this test is no longer needed as I wanted to identify the limits of localstorage.
Removed